ops/server.sh

```diff
@@ -9,6 +9,7 @@ test "$1" && export WFL_DEPLOY_ENVIRONMENT="$1"
 npm run serve --prefix=derived/ui -- --port 8080 &
 
 pushd api
+export _JAVA_OPTIONS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005"
```

Comment:
I've never used server.sh before, I think it is just for testing--I'm not quite sure what's going on here, are these options we'll want to port over into our helm stuff for prod?

Reply:
You're right, it's just for testing and I wouldn't port this to a k8s deployment (except maybe for a sandbox). This line is so you can attach a debugger and we don't wanna do this for production deployments.